### ISSUE TITLE
Infinite loop when plot is False

### ISSUE THREAD

**Issue by crond-jaist** [cesarghali/topology-generator#1]
Hi,

I found a small issue with your program, as it seems to enter an infinite loop when "plot" is set to "False" in the configuration file. The fix is actually quite simple, the code on line 363 simply needs to be shifted to the left, and for future users it would be great if you could find the time to do it.

By the way, below is the diff between the current and the fixed file:
```
363c363
<             fn = fn + 1
---
>         fn = fn + 1
```

Best wishes,
Razvan

**Comment by cesarghali** [comment]
Thank you for reporting this! Please feel free to send a PR and I'll be happy to review and approve it :)

**Comment by crond-jaist** [comment]
Just created the pull request, please have a look. Great tool, by the way!

**Comment by cesarghali** [comment]
Thanks!

Fixed by PR #2.
